frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display failure details on pipelineruns and taskruns-ODC-3713.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Add page for Pipeline
+	As a user, I want see failure details on pipelineruns and taskruns
+
+Background:
+    Given Openshift Pipeline operator is installed
+
+@regression
+Scenario: Display failure details on pipeline run details
+   Given user is at pipeline page in developer perspective
+   And a failed pipeline is present
+   When user goes to failed pipeline run
+   And user opens pipeline run details
+   Then user can see status as Failure
+   And user can view failure message under Message heading
+   And user can see Log snippet to get know what taskruns failed
+
+@regression
+Scenario: Display failure details of pipeline run in topology sidebar
+   Given user is in topology
+   And a node with failed pipeline is present
```

Comment:
Not sure how much assumption here is... but did we want to say what "node with failed pipeline" is? Like
```suggestion
   And a node with an associated pipeline that has failed is present
```